modules/simple_forms_api/spec/requests/v1/uploads_spec.rb

```diff
@@ -309,20 +309,27 @@
   end
 
   describe '#submit_supporting_documents' do
-    it 'renders the attachment as json' do
+    it 'renders the attachment as json for multiple form IDs' do
```

Comment:
Can we leave this `it` statement as it was before? I feel like the fact that it handles multiple forms is incidental to the main work under test.